charts/gitops-server/values.yaml

```diff
@@ -0,0 +1,106 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  # FIXME check the app name
+  repository: ghcr.io/weaveworks/wego-app
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.6.2"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+logLevel: info
+# Additional arguments to pass in
+# additionalArgs: []
+
+# To enable enterprise OIDC:
+# envVars:
+# - name: WEAVE_GITOPS_AUTH_ENABLED
+#   value: true
```

Comment:
this flag is also needed for the admin user login flow, just in case you wanted to note that.
i think it gates all login functionality, in core as well as ent

Reply:
Would it make more sense to turn this into a variable then, something like:
```yaml
# Sets the WEAVE_GITOPS_AUTH_ENABLED environment variable to enable login
enable_login: true
```

Reply:
yeh that sounds good